components/DiscoverFeed/DiscoverFeed.tsx

```diff
@@ -83,7 +83,7 @@ export default function DiscoverFeed() {
             itemCount={events.length}
             itemSize={getRowHeight}
             width={width}
-            overscanCount={5}
+            overscanCount={2}
```

Comment:
lowered this even more now that we're also fetching event replies on initial load to calculate the reaction counts.